apps/modernization-ui/src/apps/page-builder/page/management/edit/question/QuestionHeader.tsx

```diff
@@ -1,35 +1,68 @@
 import styles from './question-header.module.scss';
 import { Icon } from '@trussworks/react-uswds';
 import { ToggleButton } from 'apps/page-builder/components/ToggleButton';
+import { PagesQuestion } from 'apps/page-builder/generated';
+import classNames from 'classnames';
 import { Heading } from 'components/heading';
 import DeleteQuestion from '../../../../components/DeleteQuestion/DeleteQuestion';
 
 type Props = {
-    isStandard: boolean;
-    isRequired: boolean;
+    visible?: boolean;
+    question: PagesQuestion;
     onRequiredChange?: () => void;
     onEditQuestion?: () => void;
     onDeleteQuestion?: () => void;
 };
+
+const hyperlinkId = 1003;
+const commentsReadOnlyId = 1014;
+const lineSeparatorId = 1012;
+const originalElecDocId = 1036;
+const readOnlyPartId = 1030;
+
+const staticTypes = [hyperlinkId, commentsReadOnlyId, lineSeparatorId, originalElecDocId, readOnlyPartId];
+
 export const QuestionHeader = ({
-    isStandard,
-    isRequired,
+    question,
     onRequiredChange,
     onEditQuestion,
-    onDeleteQuestion
+    onDeleteQuestion,
+    visible = true
 }: Props) => {
     return (
-        <div className={styles.header}>
+        <div className={classNames(styles.header, { [styles.visible]: visible })}>
             <div className={styles.typeDisplay}>
-                {isStandard && <div className={styles.standardIndicator}>S</div>}
-                <Heading level={3}>Question</Heading>
+                {question.isStandard && <div className={styles.standardIndicator}>S</div>}
+                {staticTypes.includes(question.displayComponent!) ? (
+                    <>
+                        {question.displayComponent === hyperlinkId && (
+                            <Heading level={3}>Static element: Hyperlink</Heading>
+                        )}
+                        {question.displayComponent === lineSeparatorId && (
+                            <Heading level={3}>Static element: Line separator</Heading>
+                        )}
+                        {question.displayComponent === commentsReadOnlyId && (
+                            <Heading level={3}>Static element: Comment </Heading>
+                        )}
+                        {question.displayComponent === originalElecDocId && (
+                            <Heading level={3}>Static element: Electronic document list</Heading>
+                        )}
+                        {question.displayComponent === readOnlyPartId && (
+                            <Heading level={3}>Static element: Participant list</Heading>
+                        )}
+                    </>
+                ) : (
+                    <>
+                        <Heading level={3}>Question</Heading>
+                    </>
+                )}
```

Comment:
Suggestion: Might be cleaner to have a method that returns the appropriate heading and just call that instead of re-declaring the `<Heading ...>` element and having the if / else here.

Ex:
```typescript

    const getHeadingText = (displayComponent: number | undefined) => {
        switch (displayComponent) {
            case hyperlinkId:
                return 'Static element: Hyperlink';
            case lineSeparatorId:
                return 'Static element: Line separator';
            case commentsReadOnlyId:
                return 'Static element: Comment';
            case originalElecDocId:
                return 'Static element: Electronic document list';
            case readOnlyPartId:
                return 'Static element: Participant list';
            default:
                return 'Question';
        }
    };

.....

<Heading level={3}>{getHeadingText(question.displayComponent)}</Heading>

```